packages/next/src/build/webpack-config.ts

```diff
@@ -245,6 +259,11 @@ export function getDefineEnv({
             process.env.NEXT_EDGE_RUNTIME_PROVIDER || 'edge-runtime'
           ),
         }),
+
+    // Align with turbopack to inject turbopack process env for tree-shaking
+    'process.turbopack': JSON.stringify(
+      config.experimental.turbo ? true : undefined
+    ),
```

Comment:
```suggestion
    'process.turbopack': JSON.stringify(false),
```

This is the webpack config, so it's never using turbopack